tests/common/snappi_tests/snappi_fixtures.py

```diff
@@ -1313,10 +1313,13 @@ def static_routes_cisco_8000(addr, dut=None, intf=None, namespace=None, setup=Tr
     if dut is None:
         if addr not in DEST_TO_GATEWAY_MAP:
             raise RuntimeError(f"Request for dest addr: {addr} without setting it in advance.")
+            return addr
```

Comment:
This return logic will never be executed?

Reply:
> This return logic will never be executed?

Yes, we are addressing other platform issues here: https://github.com/sonic-net/sonic-mgmt/pull/17628